magda-web-client/src/Components/Admin/AdminHeader.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
```

Comment:
We don't really have any `jsx` files, just `js` or `tsx`, can this be one of those?